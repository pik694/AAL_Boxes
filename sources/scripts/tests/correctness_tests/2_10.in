2
35 88 73
26 89 51

