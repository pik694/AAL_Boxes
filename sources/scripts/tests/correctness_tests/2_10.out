1 0 
342854->342854
